app/models/service_template_transformation_plan_task.rb

```diff
@@ -113,16 +113,20 @@ def destination_security_group
     SecurityGroup.find_by(:id => vm_resource.options["osp_security_group_id"])
   end
 
-  def transformation_log
+  def valid_transformation_log_types
+    %w(v2v wrapper)
```

Comment:
Is the wrapper log a superset of the v2v log?  Wondering if there's any reason why someone would want to download both.

Reply:
Nope. It's a separate log. virt-v2v log is very verbose and difficult to interpret by the user. virt-v2v-wrapper log provides more comprehensive error messages when possible. When not, virt-v2v log is still a good place to look at. The support may also be interested in both when a customer opens a case.